lib/components/Masthead/Masthead.module.scss

```diff
@@ -169,7 +114,6 @@ $search-max-size: 100 * $grid-size;
             justify-content: flex-end;
             list-style-type: none;
 
-
             // hide the search-button that control the display of search-bar
             .search-button {
                 @media(min-width: $screen-md) {

```

Comment:
var

Reply:
`var(...)` can only be used as properties, therefore media query conditions cannot be set with `var(...)`... for this we have to keep using scss vars